src/components/utils/MenuItem/MenuItem.tsx

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 import { Theme } from '../../../theme';
-import Box, { NativeAttributes } from '../../Box';
+import Box, { BoxProps } from '../../Box';
 
-interface MenuItemProps extends NativeAttributes<'div'> {
```

Comment:
I'd like to keep this as is and add whatever overrides you want to a wrapper element